buildSrc/src/main/groovy/airbyte-bulk-connector.gradle

```diff
@@ -130,6 +130,26 @@ class AirbyteBulkConnectorPlugin implements Plugin<Project> {
             ]
         }
 
+        project.sourceSets {
+            integrationTestJava {
```

Comment:
You should rename this to `integrationTest`.

This `integrationTestJava` is a holdover from the bad old days where there was an `integrationTestPython` (don't ask). 

Having 'java' in the source set name results in all kinds of nasty gradle task names such as `compileIntegrationTestJavaJava`.

Reply:
Done!